.github/workflows/tf-account.yaml

```diff
@@ -22,9 +25,12 @@ jobs:
       id-token: write
       contents: write
       pull-requests: write
+
     steps:
       - name: Checkout
         uses: actions/checkout@v3
+        with:
+          fetch-depth: 2 # Needed to detect version changes when merging
```

Comment:
I can't tell if this is going to be too brittle. What if I push 5 commits as part of my branch?

Reply:
I'll change this to `0` which will fetch the full tree.